.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+    - "3.7"
+
+install:
+    - pip install poetry
+    - poetry install
+    - cp env-template .env
+script:
+    - poetry run flask db init
+    - poetry run flask db migrate
+    - poetry run flask db upgrade
```

Comment:
The tests run in a temporary database anyway. Is this meant to test if the migration scripts work?